pkgs/by-name/si/simp/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  autoPatchelfHook,
+  libxkbcommon,
+  vulkan-loader,
+  stdenv,
+  darwin,
+  wayland,
+  cargo-about,
+  makeDesktopItem,
+  dav1d,
+  libheif,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "simp";
+  version = "3.5.3";
+
+  src = fetchFromGitHub {
+    owner = "Kl4rry";
+    repo = "simp";
+    rev = "v${version}";
+    hash = "sha256-NALVHcU+NhCT5w+pn6WhP+cFGPuDhnPt+IDNHN/oqls=";
+  };
+
+  cargoHash = "sha256-Wp50ldXGy7MLQTnvZDx3LIr8jLBpkoCrh0/xPowIn54=";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
```

Comment:
This is a source package. Please use patchelf to fix small issues in the upstream build system

Reply:
Manual patchelf invocations are error-prone, and whether it's source-based or not, some patchelf appears to be necessary for this build. I don't see an actual issue here.